rizinorg/cutter
# Adjust r2 paths (Fix r2dec misssing from release package)

**Your checklist for this pull request**


**Detailed description**

Use the default paths expected by r2dec build scripts. Use meson directly instead of meson.py which planned to be removed.

**Test plan (required)**

* Tests are green
* Inspect Appveyor log and make sure r2dec and r2ghidra were compiled successfully (they are post build steps so doesn't cause build failure)
* Download the build from Appveyor and make sure r2dec and r2ghidra work.


**Closing issues**
closes #2329
